ibexa/commerce/4.2/encore/ibexa.webpack.config.manager.js

```diff
@@ -55,9 +55,18 @@ const add = ({ ibexaConfig, eZConfig, entryName, newItems }) => {
 
     config.entry[ibexaEntryName] = [...items, ...newItems];
 };
+const prepend = ({ ibexaConfig, eZConfig, entryName, newItems }) => {
+    const config = ibexaConfig ? ibexaConfig : eZConfig;
```

Comment:
as this is a new method do we need to care about accepting eZConfig as a parameter, I don't think so.